spec/facility_spec.rb

```diff
@@ -22,5 +35,163 @@
       @facility.add_service('Vehicle Registration')
       expect(@facility.services).to eq(['New Drivers License', 'Renew Drivers License', 'Vehicle Registration'])
     end
+
+    it 'can add available services to another facility' do
+      @facility_1.add_service('New Drivers License')
+      @facility_1.add_service('Vehicle Registration')
+      expect(@facility_1.services).to eq(['New Drivers License', 'Vehicle Registration'])
+
+      @facility_2.add_service('New Drivers License')
+      @facility_2.add_service('Renew Drivers License')
+      expect(@facility_2.services).to eq(['New Drivers License','Renew Drivers License'])
+    end
+  end
+
+  describe '#vehicles services' do
```

Comment:
I like that you created a describe block to house all the it blocks that describe the behavior of the register_vehicle method. I would have put that as the describe block though: `describe "#register_vehicle" do ` that way we know all those it blocks pertain to that method.